src/devices/Display/BarColor.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Describes LED colors in a Bargraph.
+    /// </summary>
+    public enum BarColor
+    {
+        /// <summary>
+        /// Disable LED.
+        /// </summary>
+        OFF = 0,
```

Comment:
With our naming convention, those public enums should be like `Off`, `Red`, `Green`, `Yellow`.